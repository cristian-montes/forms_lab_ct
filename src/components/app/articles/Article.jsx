import React from 'react';

export default function Article({title, author, description, image}){
    return (
        <figure>
            <img src={image} alt={author} height={200} width={200} />
            <div>
                <p>{title}</p>
                <p>{description}</p>
                <p> Author: {author}</p>
            </div>
        </figure>
    )
}