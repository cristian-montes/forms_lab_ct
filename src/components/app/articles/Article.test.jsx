import React, { Component } from 'react';
import { render } from '@testing-library/react';
import Article from './Article';

describe('article test', () => {
  it('should render a single article for the list', () => {
    const { asFragment } = render(
      <Article
        title="The Taco King"
        author="Taquito"
        description="I make any type of tacos you want"
      />
    );
    expect(asFragment()).toMatchSnapshot();
  });
});