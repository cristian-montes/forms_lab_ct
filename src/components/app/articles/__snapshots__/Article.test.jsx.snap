// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`article test should render a single article for the list 1`] = `
<DocumentFragment>
  <figure>
    <img
      alt="Taquito"
      height="200"
      width="200"
    />
    <div>
      <p>
        The Taco King
      </p>
      <p>
        I make any type of tacos you want
      </p>
      <p>
         Author: Taquito
      </p>
    </div>
  </figure>
</DocumentFragment>
`;
